src/index.js

```diff
@@ -196,6 +197,13 @@ function App() {
             <ParentFaq LoadingComponent={ParentLoadingComponent} />
           )}
         />
+        <SecureRoute
+          path="/parent/contact"
+          exact
+          component={() => (
```

Comment:
I don't think you need the 'exact' keyword in here as that only applies to '/' URLs - I might be mistaken about that though so - if someone else can back me up here that would be great.